### DISCUSSION TITLE
Icon Request: icon-tattoo

### DISCUSSION THREAD

**Discussion by kryptosucks** [FortAwesome/Font-Awesome#21224] [Icon Requests]
It would be awesome to have a little tattoo machine icon. This could label tattoo photos, if someone is a tattoo artist, etc.
![images-2](https://f.cloud.github.com/assets/2029956/808735/fb78a0f8-ee6a-11e2-825a-8880a94970b9.jpg)
![image](https://f.cloud.github.com/assets/2029956/808747/28292abe-ee6b-11e2-8039-bf207b86c8ff.jpg)

Thanks!

**Comment by tagliala** [comment]
is there a simpler shape for this one?

**Comment by tagliala** [comment]
@psyance a simpler shape that would fit 14x14 px?

**Comment by kryptosucks** [comment]
I'll see what I can come up with :)

.: Adam

**Comment by RazoR9944** [comment]
![ce53426a7273737986f9481984f424e1](https://cloud.githubusercontent.com/assets/13030631/8326211/d7d472ee-1a5f-11e5-9f15-3db2fa561c41.png)

![images](https://cloud.githubusercontent.com/assets/13030631/8326230/ef61ce70-1a5f-11e5-935d-74db000e611c.png)

 Something like that would be nice.
And maybe a piercing icon too. 
Thanks

**Comment by pRob3** [comment]
+1

**Comment by Netroxen** [comment]
+1

**Comment by raphaelfrota** [comment]
+1

**Comment by kjdion84** [comment]
+1

**Comment by natelaff** [comment]
+1

**Comment by Clayboy** [comment]
+1

**Comment by toth-attila** [comment]
+1

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂
